src/controller.rs

```diff
@@ -122,17 +127,23 @@ pub fn start(
                                 error!("Error response from server: {:?}", failure);
                                 if let Some(request) = ctx.response_waitlist.remove(&failure.id) {
                                     let (meta, method, _) = request;
-                                    let msg = match failure.error.code {
-                                        ErrorCode::MethodNotFound => format!(
-                                            "{} language server doesn't support method {}",
-                                            ctx.language_id, method
-                                        ),
-                                        _ => format!(
-                                            "{} language server error: {}",
-                                            ctx.language_id, editor_quote(&failure.error.message)
-                                        ),
-                                    };
-                                    ctx.exec(meta, format!("lsp-show-error {}", editor_quote(&msg)));
+                                    match failure.error.code {
+                                        ErrorCode::ServerError(CONTENT_MODIFIED) => {},
+                                        code => {
+                                            let msg = match code {
+                                                ErrorCode::MethodNotFound => format!(
+                                                    "{} language server doesn't support method {}",
+                                                    ctx.language_id, method
+                                                ),
+                                                ErrorCode::ServerError(CONTENT_MODIFIED) => String::new(),
```

Comment:
Sorry for nitpicking, this line is redundant now =)

Reply:
Oops, sorry...